test/planning/planner.test.ts

```diff
@@ -493,6 +493,16 @@ describe("Planner", () => {
         container.get(Test2);
     });
 
+    it("Should ignore checking base classes for @injectable on resolve when skipBaseClassChecks is set", () => {
```

Comment:
You didn't talk about this in your PR comments. Was this just a test case that you saw was missing?

Reply:
This test case cover the issue #914